File: Pokua/BasicChallenges/Exercise26.java
package BasicChallenges;

import java.util.Scanner;

public class Exercise26 {

    public void primeChecker(int n){
        boolean isPrime = true;

        if (n <= 1) {
            isPrime = false;
        }

        for (int i = 2; i <= Math.sqrt(n); i++) {
            System.out.println(i);
            if (n % i == 0) {
                isPrime = false;
                break;
            }
        }

        if (isPrime) {
            System.out.println("Prime");
        } else {
            System.out.println("Not Prime");
        }
    }

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Enter a number: ");
        int num = input.nextInt();

        Exercise26 obj = new  Exercise26();

        obj.primeChecker(num);

        input.close();
    }
}

File: Stephannie/Exercise-26.py
# prime checker
import math
def P_check(number):
    if number <= 1:
        print(f"{number} is not prime.")
        return False
    if number == 2:
        print(f"{number} is a prime number.")
        return False
    if number % 2 == 0:
        print(f"{number} is not prime.")
        return False
    for i in range(3, int(math.sqrt(number)), 2):
        if number % i == 0: 
            print(f"{number} is not a prime number.")
            return False
    print(f"{number} is a prime number.")
    return True
P_check(27)


    
